docs/recipes/install/rocky9/x86_64/openchami/slurm/recipe.sh

```diff
@@ -0,0 +1,587 @@
+
+inputFile=${OHPC_INPUT_LOCAL:-/opt/ohpc/pub/doc/recipes/rocky9/input.local}
+
+if [ ! -e ${inputFile} ];then
+   echo "Error: Unable to access local input file -> ${inputFile}"
+   exit 1
+else
+   . ${inputFile} || { echo "Error sourcing ${inputFile}"; exit 1; }
+fi
+
+# ---------------------------- Begin OpenHPC Recipe ---------------------------------------
+# Commands below are extracted from an OpenHPC install guide recipe and are intended for
+# execution on the master SMS host.
+# -----------------------------------------------------------------------------------------
+
+# Disable firewall
+systemctl disable firewalld
+systemctl stop firewalld
+
+
+# ------------------------------
+# Enable OpenCHAMI repositories 
+# ------------------------------
+PI_URL="https://api.github.com/repos/openchami/release/releases/latest"release_json=$(curl -s "$API_URL")
+key_url=$(echo "$release_json" | jq -r '.assets[] | select(.name | endswith(".asc")) | .browser_download_url' | head -n 1)
+key_name=$(echo "$release_json" | jq -r '.assets[] | select(.name | endswith(".asc")) | .name' | head -n 1)
+curl -L -o "$key_name" "$key_url"
+rpm --import "$key_name"
+
+# --------------------------------
+# Creating OpenCHAMI certificates 
+# --------------------------------
+echo "${sms ip} ${sms name}" | sudo tee -a /etc/hosts > /dev/null
+systemctl start step-ca
+podman run --rm --network openchami-cert-internal docker.io/curlimages/curl -sk https://step-ca:9000/roots.pem | sudo tee /etc/pki/ca-trust/source/anchors/ochami.pem
+update-ca-trust
+
+
+# -----------------------------------------------
+# Complete basic Confluent setup for master node 
+# -----------------------------------------------
+mkdir -p /opt/workdir
+chown -R $USER: /opt/workdir
+cat > /etc/containers/systemd/registry.container <<EOF
+# registry.container
+[Unit]
+Description=Image OCI Registry
+After=network-online.target
+Requires=network-online.target
+[Container]
+ContainerName=registry
+HostName=registry
+Image=docker.io/library/registry:latest
+Volume=/data/oci:/var/lib/registry:Z
+PublishPort=5000:5000
+[Service]
+TimeoutStartSec=0
+Restart=always
+[Install]
+WantedBy=multi-user.target
+EOF
+cat > /etc/containers/systemd/minio.container <<EOF
+[Unit]
+Description=Minio S3
+After=local-fs.target network-online.target
+Wants=local-fs.target network-online.target
+[Container]
+ContainerName=minio-server
+Image=docker.io/minio/minio:latest
+# Volumes
+Volume=/data/s3:/data:Z
+# Ports
+PublishPort=9000:9000
+PublishPort=9091:9001
+# Environemnt Variables
+Environment=MINIO_ROOT_USER=admin
+Environment=MINIO_ROOT_PASSWORD=password1234
+# Command to run in container
+Exec=server /data --console-address :9001
+[Service]
+Restart=always
+[Install]
+WantedBy=multi-user.target
+EOF
+
+# Define local domainname, deployment protocol and dns
+systemctl daemon-reload
+systemctl enable minio.service
+systemctl enale registry.service
```

Comment:
enabling those two services fails with:
`Failed to enable unit: Unit /run/systemd/generator/minio.service is transient or generated.`